packages/cascara/src/modules/DataTextArea/DataTextArea.doc.mdx

```diff
@@ -22,9 +22,9 @@ Feel free to play around with the following code:
   }}
 >
   <DataTextArea
-    attribute='firstName'
+    attribute='Textarea'
     isEditable={true}
-    label='First Name'
+    label='Textarea'
```

Comment:
```suggestion
    label='TextArea'
```
I think we should consider trying to keep some consistency with these labels and the components themselves. This probably means we should remove "Data" from the component name and use that as the label.

Reply:
We have Data in all our module input components.

Reply:
I am talking specifically about the component name "DataTextArea" removing "Data" so the label prop is just "TextArea" with the same capitalization.

Reply:
Done